java-client/src/main/java/org/opensearch/client/opensearch/_types/analysis/Analyzer.java

```diff
@@ -589,8 +592,11 @@ public Analyzer build() {
 
     protected static void setupAnalyzerDeserializer(ObjectDeserializer<Builder> op) {
 
+        for (Language value : Language.values()) {
+            op.add(Builder::language, LanguageAnalyzer._DESERIALIZER, value.jsonValue().toLowerCase());
+            //TODO should we lowercase in the Language Enum?
```

Comment:
Language jsonValue has to match the Kind jsonValue
So we set the deserializers for all the languages here